From d59baf2dbfff765a559522f71d084de23542bcb4 Mon Sep 17 00:00:00 2001
From: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Date: Fri, 18 Jan 2019 04:27:07 +0200
Subject: [PATCH 013/269] s32g: Enable RTC interrupt

Declare RTC IRQ for properly routing after the system is woken up.

In order to resume the suspended system, the RTC will first generate a
wakeup event with the MC_WKPU, then raise an interrupt which will have
to be cleared by the core.

Upstream-Status: Pending 

Signed-off-by: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
Issue: ALB-3452
---
 plat/s32g/s32g275_bl31.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/plat/s32g/s32g275_bl31.c b/plat/s32g/s32g275_bl31.c
index 0ec6e95f1..692144b25 100644
--- a/plat/s32g/s32g275_bl31.c
+++ b/plat/s32g/s32g275_bl31.c
@@ -83,6 +83,8 @@ static uintptr_t rdistif_base_addrs[PLATFORM_CORE_COUNT];
 static const interrupt_prop_t interrupt_props[] = {
 	INTR_PROP_DESC(29, GIC_HIGHEST_SEC_PRIORITY,
 		       INTR_GROUP0, GIC_INTR_CFG_EDGE),
+	INTR_PROP_DESC(153, GIC_HIGHEST_NS_PRIORITY,
+		       INTR_GROUP1S, GIC_INTR_CFG_LEVEL),
 };
 
 const gicv3_driver_data_t s32g275_gic_data = {
-- 
2.17.1

